components/example-notebook-servers/codeserver-python/Dockerfile

```diff
@@ -1,15 +1,16 @@
-FROM public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/codeserver:master-ebc0c4f0
+FROM public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/codeserver:master-434b10ab
 
 USER root
 
 # args - software versions
-ARG CODESERVER_PYTHON_VERSION="2021.3.658691958"
-ARG CONDA_VERSION="4.9.2"
+ARG CODESERVER_PYTHON_VERSION=2021.5.842923320
+ # renovate: datasource=github-tags depName=conda/conda versioning=semver
+ARG CONDA_VERSION=4.10.1
```

Comment:
Lets remove the `CONDA_VERSION` arg, as if it is updated independent of `MINIFORGE_VERSION` there will be issues.

Reply:
We can use bash variable parameter expansion to remove the `-4` suffix: https://wiki.bash-hackers.org/syntax/pe

Reply:
Resolved, using `${MINIFORGE_VERSION:0:-2}` now instead.